tasks/config/intern.js

```diff
@@ -8,6 +8,12 @@ module.exports = function (grunt) {
       config: 'test/intern',
       reporters: ['Console']
     },
-    dev: {}
+    dev: {},
+    api: {
```

Comment:
Yeah I'll probably refactor these when I have some more time. They don't start a browser since the intern classifies them as unit tests, but in hindsight they definitely don't need to rely on the intern at all.